src/sdl_utils.cpp

```diff
@@ -0,0 +1,57 @@
+#include "sdl_gl.h"
+#include <iostream>
+
+void sdl_application_abort(const char * msg)
+{
+    std::cout << SDL_GetError() << std::endl;
+    SDL_Quit();
+    exit (-1);
+}
+
+void getInfo(void)
+{
+    SDL_version compiled;
+    SDL_version linked;
+    SDL_VERSION(&compiled);
+    SDL_GetVersion(&linked);
+
+    fprintf(stdout, "We compiled against SDL version %d.%d.%d ...\n",compiled.major, compiled.minor, compiled.patch);
+    fprintf(stdout, "And we are linking against SDL version %d.%d.%d.\n", linked.major, linked.minor, linked.patch);
+
+    int glMajor = 0;
+    int glminor = 0;
+    glGetIntegerv(GL_MAJOR_VERSION, &glMajor);
+    glGetIntegerv(GL_MINOR_VERSION, &glminor);
+    std::cout << "Current OpenGL version : " << glMajor << "." << glminor << std::endl;
+}
+
+void getTextureInfo( SDL_Texture * texture)

```

Comment:
How about: "PrintSDLTextureInfo"?
No space before first parameter.
I know everyone has their own style about declaring pointers:
Var *name;
Var* name;
Var * name;
etc. I guess I won't nit pick this, at least it's self consistent in this file. It just differs from other files.